content/400-reference/100-system-requirements.mdx

```diff
@@ -21,7 +33,28 @@ Notes:
 
 See also: [Supported database versions](database-reference/supported-databases)
 
-</TopBlock>
+### Operating systems
+
+Prisma is supported on MacOS, Windows and most Linux distributions.
+
+#### Linux runtime dependencies
+
+Prisma requires the following system libraries installed to work:
+
+- OpenSSL 1.0.x, 1.1.x or 3.0.x
+- On glibc-based distributions, glibc 2.17 or newer (glibc 2.24 or newer on ARM64)
+- zlib (`libz.so.1`)
+- libgcc (`libgcc_s.so.1`)
```

Comment:
`Libgcc` instead maybe?
https://gcc.gnu.org/onlinedocs/gccint/Libgcc.html

(Unfortunately not consistent: https://gcc.gnu.org/gcc-3.0/libgcc.html)

Reply:
I think lowercase would be more standard/widespread

Reply:
no strong opinion though